compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaPathParameterSymbol.java

```diff
@@ -49,10 +49,13 @@ public class BallerinaPathParameterSymbol extends BallerinaSymbol implements Pat
     private List<AnnotationSymbol> annots;
     private List<io.ballerina.compiler.api.symbols.AnnotationAttachmentSymbol> annotAttachments;
     private String signature;
+    private boolean isTypeOnlyPathParam;
 
-    public BallerinaPathParameterSymbol(PathSegment.Kind kind, BSymbol symbol, CompilerContext context) {
+    public BallerinaPathParameterSymbol(PathSegment.Kind kind, BSymbol symbol, CompilerContext context,
```

Comment:
Are we ok to change public functions in patch branches?

Reply:
Will be only changed in the master at: #38146.